packages/playground/blueprints/src/lib/steps/enable-multisite.ts

```diff
@@ -34,6 +29,8 @@ export interface EnableMultisiteStep {
 export const enableMultisite: StepHandler<EnableMultisiteStep> = async (
 	playground
 ) => {
+	await installWpCli(playground);
```

Comment:
This will always fetch wp-cli, even when it's already available.

Reply:
I forgot to add that part back after making changes. It's fixed now.